docs(api): PD interop changes in 8.5; new params (#18686)
OpenTrons/opentrons-api: api/docs/v2/basic_commands/liquids.rst

    pipette.aspirate(200, plate["A1"], rate=2.0)

> You can also specify an absolute ``flow_rate`` to change the default flow rate for your pipette::

Reviewer: Saying "default" here makes it sound like changing `InstrumentContext.flow_rates.aspirate` — the change will apply going forward. But it's actually just for this one command. I think this is clearer:
```suggestion
You can also specify an absolute ``flow_rate`` to set the flow rate in µL/second::
```

    pipette.aspirate(200, plate["A1"], flow_rate=50)

> The ``rate`` and ``flow_rate`` parameters are mutually exclusive. If you choose to change the ``flow_rate``, specifying a ``rate`` will raise an error. 

Reviewer: Just a suggestion.
```suggestion
The ``rate`` and ``flow_rate`` parameters are mutually exclusive. If you specify both in the same command, the API will raise an error. 
```

Author: cleaner and better! done!

.. versionadded:: 2.0

> .. versionchanged:: 2.24

Reviewer: Alternative: move the 2.0 notice under the first code block and change this to a version added notice directly under the second code block.

    pipette.dispense(200, plate["B1"], rate=2.0)

You can also specify an absolute ``flow_rate`` to change the default flow rate for your pipette::

    pipette.dispense(200, plate["B1"], flow_rate=50)

The ``rate`` and ``flow_rate`` parameters are mutually exclusive. If you choose to change the ``flow_rate``, specifying a ``rate`` will raise an error.  

.. versionadded:: 2.0
.. versionchanged:: 2.24

>     Add the dispense ``flow_rate`` parameter. 

Reviewer: Replicate any changes made for aspirate.

This example demonstrates touching the tip in a specific well::

    pipette.touch_tip(plate["B1"])

.. versionadded:: 2.0
    
This example uses an offset to set the touch tip location 2mm below the top of the current well::

    pipette.touch_tip(v_offset=-2) 

This example moves the pipette 75% of well's total radius and 2 mm below the top of well::

    pipette.touch_tip(plate["B1"], radius=0.75, v_offset=-2)

And this example uses ``mm_from edge`` to set the touch tip location 0 mm, or the edge of the current well::

>     pipette.touch_tip(plate["B1"],mm_from_edge=0)

Reviewer 2: Space after the comma please.

Author: Good catch; thank you!!

                      radius=0.75,
                      v_offset=-2)

And this example uses ``mm_from edge`` to set the touch tip location 0 mm, or the edge of the current well::

    pipette.touch_tip(plate["B1"],

>                       mm_from_edge=0)

Reviewer: The indentation here, and in the existing example above, are weird. They're both [short enough to fit on one line](https://black.vercel.app/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ADiAH5dAD2IimZxl1N_WmOKu8X3q5HjiOJYmGPeIZG_lqnu0fhWIwI0AB81AIYWeBNSInq2Ff0kU3ZI3VZ4JsvFcL3KzhpK81b3Ui5T3S8xwLUoiLCuwoxoycBlONctz_XRulRBviWmvheFGw6KsGOYWu_B1bY9l87xKGXkRtXb3Lg0YAAAACShtBFm2_zWAAGaAeMBAACpTMCHscRn-wIAAAAABFla).

.. versionchanged:: 2.4
    Lowered minimum speed to 1 mm/s.

> .. versionchanged:: 2.24

Reviewer: As this section (Touch Tip) gets longer, it's hard to tell that this goes with the whole thing. Could move:
- 2.0 after first snippet `pipette.touch_tip()`
- 2.4 here, at end of Touch Speed §
- 2.24 after snippet `pipette.touch_tip(plate["B1"], mm_from_edge=0)`

Mix
====

> The :py:meth:`~.InstrumentContext.mix` method aspirates and dispenses repeatedly in a single location. It's designed to mix the contents of a well together using a single command rather than using multiple ``aspirate()`` and ``dispense()`` calls. This method includes arguments that let you specify the number of times to mix, the volume (in µL) of liquid, and the well that contains the liquid you want to mix. 

Reviewer: Spurious change, and better to not have trailing whitespace.

    pipette.mix(repetitions=3)

Like an ``aspirate()`` or ``dispense()``, you can use optional arguments to specify the flow rate, a delay, or a push out after an aspirate or dispense in the mix. 

> This example draws 100 µL from the current well and mixes it three times, with an ``aspirate_flow_rate`` of 50 µL/sec and a 5 second delay after each aspirate::

Reviewer: Better to explain than recap param name.
```suggestion
This example draws 100 µL from the current well and mixes it three times, aspirating at 50 µL/sec and with a 5 second delay after each aspirate::
```

    pipette.mix(repetitions=3)

Like an ``aspirate()`` or ``dispense()``, you can use optional arguments to specify the flow rate, a delay, or a push out after an aspirate or dispense in the mix. 

This example draws 100 µL from the current well and mixes it three times, aspirating at 50 µL/sec and with a 5 second delay after each aspirate::

    pipette.mix(
        repetitions=3,
        volume=100,
        aspirate_flow_rate=50,
        aspirate_delay=5
    )

And this example adds a push out of 10 µL after the final dispense in the mix::

>     pipette.mix(repetitions=3, volume=100, final_push_out=10)

Reviewer 2: It may be worth syncing with Ed or the engineering team about how much we want to say about push-out. It's kind of subtle and tricky.

Reviewer: I think this level of detail is consistent with what we say about push out with dispense.

Reviewer 2: That's fair.

I think the part that makes me slightly nervous is that the text says `And this example **adds** a push out of 10 uL`. That implies, to a naive reader, that if the example **hadn't** had `final_push_out=10`, there would be no final push-out at all, which is false. (If the user doesn't specify the `final_push_out`, we would have added some unspecified final push-out amount, consistent with the behavior before this argument was introduced.)

Author: Leaving this one for now because we cover push outs in detail earlier on the page. Happy to come back to it later!

Air Gap
=======

> The :py:meth:`.InstrumentContext.air_gap` method tells the pipette to draw in air before or after a liquid. Creating an air gap helps keep liquids from seeping out of a pipette after drawing it from a well. This method includes arguments that give you control over the amount of air to aspirate and the position at the target well to add the air gap. By default, the pipette moves 5 mm above a well before aspirating air. Calling :py:meth:`~.InstrumentContext.air_gap` with no arguments uses the entire remaining volume in the pipette.

Reviewer: Maybe "5 mm above the center of a well", since there's now xyz control over location.
